lp-simulation-environment/monitoring/src/main/java/eu/learnpad/simulator/api/impl/utils/RestResource.java

```diff
@@ -0,0 +1,40 @@
+package eu.learnpad.simulator.api.impl.utils;
+
+import org.apache.commons.httpclient.Credentials;
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.UsernamePasswordCredentials;
+import org.apache.commons.httpclient.auth.AuthScope;
+
+public class RestResource {
+	final private static String PROTOCOL = "http";
+	final private static String HOSTNAME = "localhost";
+	final private static int PORT = 8080;
+	final private static int SIM_PORT = 8081;
+	final private static String HOST = PROTOCOL + "://" + HOSTNAME + ":" + PORT;
+	final private static String SIM_HOST = PROTOCOL + "://" + HOSTNAME + ":" + SIM_PORT;
+	final private static String DEFAULT_USER = "userName";
+	final private static String DEFAULT_PASSWORD = "LearnPAss";
+	final public static String REST_URI = HOST + "/sim/mon/rest";
```

Comment:
If this is the Core Platform you want to contact, the prefix is `/xwiki/rest/` and you may have to append `/learnpad/sim/corefacade` for all the REST services for the Controller of Simulation Environment.